pkg/cluster/install.go

```diff
@@ -52,6 +52,9 @@ func (m *manager) adminUpdate() []steps.Step {
 		toRun = append(toRun,
 			steps.AuthorizationRefreshingAction(m.fpAuthorizer, steps.Action(m.ensureResourceGroup)), // re-create RP RBAC if needed after tenant migration
 			steps.Action(m.createOrUpdateDenyAssignment),
+			steps.AuthorizationRefreshingAction(m.fpAuthorizer, steps.Action(m.enableServiceEndpoints)),
```

Comment:
If the Operator does it, do we strictly need to do this in the AdminUpdate?

Reply:
This isn't necessarily needed, but it may eliminate some errors upon enabling the storage lockdown on the controller for existing clusters.  

Over time, desired state would look good, but if we removed this step during the first controller run and the controller which sets storage account virtual network rules attempts to run first, the storage account will fail to update because storage endpoints must first be set on the subnet.  

This will stop an SRE from wondering if this is a normal error with the controller, or if it's a first time reconciliation issue.